packages/web-app-files/src/views/spaces/GenericSpace.vue

```diff
@@ -333,6 +320,43 @@ export default defineComponent({
       )
     })
 
+    const focusAndAnnounceBreadcrumb = (sameRoute) => {
+      const breadcrumbEl = document.getElementById('files-breadcrumb')
+      if (!breadcrumbEl) {
+        return
+      }
+      const activeBreadcrumb = last(breadcrumbEl.children[0].children)
+      const activeBreadcrumbItem = activeBreadcrumb.getElementsByTagName('button')[0]
+      if (!activeBreadcrumbItem) {
+        return
+      }
+
+      const totalFilesCount = store.getters['Files/totalFilesCount']
+      const itemCount = totalFilesCount.files + totalFilesCount.folders
+
+      const announcement = $gettextInterpolate(
```

Comment:
`$gettextInterpolate` is not needed anymore. Interpolation can happen in `$gettext` and `$ngettext`